jvm/src/main/kotlin/app/cash/trifle/internal/validators/X509CertChainValidator.kt

```diff
@@ -42,7 +47,7 @@ internal class X509CertChainValidator(certAnchor: Certificate) : CertChainValida
       PATH_VALIDATOR.validate(X509FACTORY.generateCertPath(x509Certs), pkixParams)
       true
     } catch (e: Exception) {
-      false
+      throw e
```

Comment:
If we’re just rethrowing the exception we don’t need to catch it

Reply:
right, right.  good catch (pun intended?)